src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIO.java

```diff
@@ -48,7 +52,4 @@ public default void setLeftVelocitySetpoint(double velocityRadiansPerSecond) {}
 
   /** Sets motor closed loop velocity setpoint for right flywheel. */
   public default void setRightVelocitySetpoint(double velocityRadiansPerSecond) {}
```

Comment:
Because the motion profile sets each setpoint, we should probably rename this to ```setLeftVelocityGoal``` instead.